src/main/java/app/controllers/UserController.java

```diff
@@ -54,7 +54,7 @@ public static void login(Context ctx, ConnectionPool connectionPool) {
             ctx.sessionAttribute("currentUser", user);
             // Hvis ja, send videre til forsiden med login besked
             ctx.attribute("message", "Du er nu logget ind");
-            ctx.render("index.html");
+            ctx.render("user-frontpage.html");
```

Comment:
Should it route to index.html and tell the user their account has been created and that they can login? It's unclear if a "You're now logged in" message is desired on the user-frontpage